prusti-viper/src/encoder/encoder.rs

```diff
@@ -1421,7 +1421,9 @@ impl<'v, 'tcx> Encoder<'v, 'tcx> {
         proc_def_id = *self.get_specification_def_id(&proc_def_id);
         // The stub function may come from another module for which we can have
         // only optimized_mir.
+        //let body = self.env.tcx().optimized_mir(proc_def_id); // FIXME: https://github.com/viperproject/prusti-dev/issues/286
```

Comment:
Have you checked what happens when you do use the `optimized_mir`? Do you also get errors about the MIR being stolen? (That is something I ran into when working on #246)